deploy/docker/zeppelin/zeppelin-site.xml

```diff
@@ -0,0 +1,105 @@
+<!--
+  Hoptimator specific settings, modified from /opt/zeppelin/conf/zeppelin-site.xml.template.
```

Comment:
Need to add the Apache copyright header to this file.

Reply:
Nice catch.